hardware/tftp/Android.mk

```diff
@@ -18,25 +18,41 @@ target_prefixes := \
     slpi \
     cdsp
 
+target_soc := \
+    msm
+
+ifeq ($(TARGET_USES_ESOC),true)
+target_soc += \
+    mdm
+endif
+
 # Prepend vendor and prefix directory to all link names:
-SONY_SYMLINKS := $(foreach prefix,$(target_prefixes), \
-    $(foreach s,$(target_combinations), \
-        $(eval p := $(subst :,$(space),$(s))) \
-        $(word 1,$(p)):$(TARGET_COPY_OUT_VENDOR)/rfs/msm/$(prefix)/$(word 2,$(p)) \
-    ) \
-)
+SONY_SYMLINKS := \
+    $(foreach tgtsoc,$(target_soc), \
+        $(foreach prefix,$(target_prefixes), \
+            $(foreach s,$(target_combinations), \
+                $(eval p := $(subst :,$(space),$(s))) \
+                $(word 1,$(p)):$(TARGET_COPY_OUT_VENDOR)/rfs/$(tgtsoc)/$(prefix)/$(word 2,$(p)) \
+            ) \
+        )
+    )
 
 # Edgecase for readwrite folders that all point to their own persist folder:
-SONY_SYMLINKS += $(foreach prefix,$(target_prefixes), \
-    /mnt/vendor/persist/rfs/msm/$(prefix):$(TARGET_COPY_OUT_VENDOR)/rfs/msm/$(prefix)/readwrite \
-)
+SONY_SYMLINKS += \
+    $(foreach tgtsoc,$(target_soc), \
+        $(foreach prefix,$(target_prefixes), \
+            /mnt/vendor/persist/rfs/msm/$(prefix):$(TARGET_COPY_OUT_VENDOR)/rfs/msm/$(prefix)/readwrite \
+        )
```

Comment:
```suggestion
        ) \
```

And here :)

Reply:
All of the missed '\' have been solved yesterday already....

Reply:
Fun fact, these suggestions were posted before that!

But for some reason I am not allowed to mark _my own comments_ as resolved. Send the blame to GH!